code/modules/mapping/mapping_helpers.dm

```diff
@@ -230,3 +230,71 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 				qdel(part)
 		j.update_icon()
 	qdel(src)
+
+
+//On Ian's birthday, the hop's office is decorated.
+/obj/effect/mapping_helpers/ianbirthday
+	name = "Ian's Bday Helper"
+	late = TRUE
+	icon_state = "iansbdayhelper"
+	var/balloon_clusters = 2
+	var/delete_after = FALSE
+
+/obj/effect/mapping_helpers/ianbirthday/LateInitialize()
```

Comment:
why not just in init? the items will all exist at that piont

Reply:
does not work